lib/emque/producing/publisher/rabbitmq.rb

```diff
@@ -5,6 +5,9 @@ module Emque
   module Producing
     module Publisher
       class RabbitMq < Emque::Producing::Publisher::Base
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Bunny::Exception
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Timeout::Error
```

Comment:
I'm also not thrilled by the constant manipulation across namespaces. Could we make this a configurable via class methods instead of the constant?